README.md

```diff
@@ -195,7 +195,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 **[⬆ back to top](#table-of-contents)**
 
 ## APIs, Data and ML
-
+  * [Random Stuff API](https://rapidapi.com/pgamerxdev/api/random-stuff-api) — Free AI chat bot API for discord bots
```

Comment:
It's fine and all but I dont think its useful for the purpose of the list, so think I'll pass thank you